specification/batch/data-plane/Microsoft.Batch/stable/2022-10-01.16.0/BatchService.json

```diff
@@ -16250,7 +16308,7 @@
         "allowTaskPreemption": {
           "type": "boolean",
           "title": "Whether Tasks in this job can be preempted by other high priority jobs",
-          "description": "If the value is set to True, other high priority jobs submitted to the system will take precedence and will be able requeue tasks from this job. You can update a job's allowTaskPreemption after it has been created using the update job API."
+          "description": "If the value is set to True other high priority jobs submitted to the system will take precedence and will be able requeue tasks from this job. You can update a job's allowTaskPreemption after it has been created using the update job API."
```

Comment:
Add back the comma

Reply:
I think you might not have your PR refreshed, thats was fixed already